src/StoreApi/Schemas/V1/CartSchema.php

```diff
@@ -377,19 +377,20 @@ public function get_item_response( $cart ) {
 	 */
 	protected function get_totals( $cart ) {
 		$has_calculated_shipping = $cart->show_shipping();
+		$decimals                = wc_get_price_decimals();
```

Comment:
Small improvement prevents the function call and filter running multiple times in this 1 method.

Reply:
Nice, let's try to find more improvements like this, I'm sure there are a lot of places we can make minor gains.